### PR TITLE
feat: Add nft medium type to filter

### PR DESCRIPTION
**Pull request by annacarey**
[PX-4828](https://artsyproduct.atlassian.net/browse/PX-4828)

This PR adds "NFT" to the artwork grid filter (I believe also used on fair page and collections page) behind an admin flag.

Also adds "NFT" to `createCollectUrl`.

I guess this needs gravity/crispr? work to actually show works marked as "NFT"? Am I missing something in Force?

Admin:
Screen Shot 2022-01-12 at 9 57 15 AM

Non admin:
Screen Shot 2022-01-12 at 9 48 57 AM

Checked:
Screen Shot 2022-01-12 at 6 43 52 PM

### CODE REVIEW

File: src/v2/Components/ArtworkFilter/ArtworkFilters/MediumFilter.tsx [hunk 1]
```diff
@@ -26,8 +29,13 @@ export const MediumFilter: FC<MediumFilterProps> = ({ expanded }) => {
     slice: "",
     counts: [],
   }
+
+  const filteredHardcodedMediums = userIsAdmin(user)
+    ? hardcodedMediums
+    : hardcodedMediums.filter(medium => medium.value !== "nft")
```
Comment (anandaroop) [inline src/v2/Components/ArtworkFilter/ArtworkFilters/MediumFilter.tsx:35]: 👍🏽

**Review by anandaroop — DISMISSED** [review]
> I guess this needs gravity/crispr? work to actually show works marked as "NFT"? Am I missing something in Force?


Yes, I'd expect that^ to be the case. You'll only show matching results once there are artworks that have the NFT gene applied (which maybe doesn't exist yet?)

**Comment by anandaroop** [comment]
Oh, and even in the absence of any actual gene automations, you could use Rosalind to batch apply the NFT gene to a bunch of artworks just to be able to see some results — this might make your Hackathon target more doable than messing with Crispr.
batch

**Comment by annacarey** [comment]
@anandaroop "NFT" isn't a gene I don't believe, so I'm going to talk to Arielle about adding it. But yes this is a good backup, thanks for surfacing!

File: src/v2/Apps/Artwork/Utils/createCollectUrl.tsx [hunk 2]
```diff
@@ -63,4 +63,5 @@ const filterCategories = {
   "Textile Arts": "textiles",
   "Video/Film/Animation": "film-slash-video",
   "Work on Paper": "work-on-paper",
+  NFT: "nft",
```
Comment (damassi) [inline src/v2/Apps/Artwork/Utils/createCollectUrl.tsx:66]: #tinynit - alpa pls

File: src/v2/Components/ArtworkFilter/ArtworkFilters/MediumFilter.tsx [hunk 3]
```diff
@@ -9,13 +9,16 @@ import { intersection } from "lodash"
...
+import { userIsAdmin } from "v2/Utils/user"
+import { SystemContext } from "v2/System"
 
 export interface MediumFilterProps {
   expanded?: boolean
 }
 
 export const MediumFilter: FC<MediumFilterProps> = ({ expanded }) => {
   const { aggregations, counts, ...filterContext } = useArtworkFilterContext()
+  const { user } = useContext(SystemContext)
```
Comment (damassi) [inline src/v2/Components/ArtworkFilter/ArtworkFilters/MediumFilter.tsx:21]: Can use `useSystemContext` hook

**Review by damassi — DISMISSED** [review]
👌 👌

File: src/v2/Apps/Artwork/Utils/createCollectUrl.tsx (hunk 2 above)
Comment (annacarey) [reply src/v2/Apps/Artwork/Utils/createCollectUrl.tsx:66]: what does alpa mean?

Comment (annacarey) [reply src/v2/Apps/Artwork/Utils/createCollectUrl.tsx:66]: oh alphabetical! you got it!

**Pull request merged**